# YAML file that provides parameters for tube_world simulation
tube_radius: 0.0762
robot_radius: 0.095
tube1_location: [1.0, 1.0]
tube2_location: [-1.0, -1.0]
tube3_location: [2.0, 2.0]
tube4_location: [-2.0, -2.0]
tube5_location: [-1.5, 1.5]
tube6_location: [1.5, -1.5]

sens_covMat: [0.1, 0.0, 0.0, 0.1]
tube_var: 0.001
max_range: 3.5
twist_noise: 0.0
slip_min: 0.0
slip_max: 0.0